### DISCUSSION TITLE
Did v2 get shut off?

### DISCUSSION THREAD

**Discussion by DemiImp** [etsy/open-api#562] [Q&A]
I've suddenly started getting errors for v2:

`oauth_problem=consumer_key_refused`

Did v2 get shut off without any warning?

**Comment by kievins** [comment]
We are seeing oauth_problem=token_revoked for V2 endpoints for majority of users. This started happening at around 19:00 ET.

**Comment by Gareth064** [comment]
See #561

**Comment by Gareth064** [comment]
V2 is still live. For me, I had to generate new tokens to get it working again.

**Comment by nateswart** [comment]
v2 is still up and running. There was a different issue that I'll track for everyone in #561

**Comment by DemiImp** [comment]
Thanks everyone.  It looks like some tokens are still broken despite yesterday's fix.

**Comment by rkistaps** [comment]
Hello, we are seeing auth outage all over again for both v3 and v2.
